ogc/wmts-layer/src/main/java/com/esri/samples/wmts_layer/WmtsLayerSample.java

```diff
@@ -60,15 +60,15 @@ public void start(Stage stage) {
       // create a WMTS service from a URL
       String serviceURL = "https://sampleserver6.arcgisonline.com/arcgis/rest/services/WorldTimeZones/MapServer/WMTS";
       wmtsService = new WmtsService(serviceURL);
-      wmtsService.addDoneLoadingListener(() -> {
-        if (wmtsService.getLoadStatus() == LoadStatus.LOADED) {
+      wmtsService.loadStatusProperty().addListener((observable, oldValue, newValue) -> {
```

Comment:
I'm pretty sure we need the addDoneLoadingListener here - if not, it seems possible the wmtsService load status return could be "not loaded" and trigger the warning on line 71. since it's not on a new thread checking for the load having complete. Similar thoughts to above really!

Reply:
Alerts now trigger only if load status is `FAILED_TO_LOAD`.

Reply:
The first status I obtain from the property listener is `LOADING`. I am unsure if the `NOT_LOADED` status refers to something going wrong during loading or because nothing has triggered a load (or both?). If initializing the `wmtsService` variable always triggers the load, then I don't know the difference between `NOT_LOADED` and `FAILED_TO_LOAD`. This is why I used both to trigger the warning.

Reply:
Short answer: `NOT_LOADED` should refer to nothing has triggered a load. The `.loadAsync()` call on the `wmtsService` on line 76 triggers the load (not the variable being initialized), and `FAILED_TO_LOAD` would trigger if something went wrong during loading 😃 

Long answer:
From the API reference: https://developers.arcgis.com/java/api-reference/reference/com/esri/arcgisruntime/loadable/LoadStatus.html#NOT_LOADED

> FAILED_TO_LOAD
> The object failed to load and some functionality may not work.

> NOT_LOADED
> The object is not loaded and some functionality may not work.

A failed to load response would mean there had been an error that had stopped the load from completing successfully: I imagine getting a `NOT_LOADED` response would be the default status if a load operation hadn't been called on the object. So throwing an alert that gives the message "Failed to load" when handling both cases of "failed to load" and "not loaded" isn't appropriate since the two have slightly different reasons for their behaviour. 

You'll get the first status of `LOADING` since there is a `loadAsync` call on line 76. If you removed that for your own testing purposes, you should see the NOT_LOADED status?

Reply:
`NOT_LOADED` is the status before `loadAsync` has been called. Once you have called `loadAsync` it will go to `LOADING` and then to `LOADED` or `FAILED_TO_LOAD`. Since the property starts in `NOT_LOADED` no change event will happen until it moves to a different state so this code looks ok to me.